src/test/java/org/fundacionjala/coding/ovidio/HighestAndLowestTest.java

```diff
@@ -0,0 +1,75 @@
+package org.fundacionjala.coding.ovidio;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by OvidioMiranda on 6/12/2017.
+ */
+public class HighestAndLowestTest {
+    /**
+     *
+     */
+    @Test
+    public void someTest() {
+        assertEquals(HighestAndLowest.highAndLow("4 5 29 54 4 0 -214 542 -64 1 -3 6 -6"), "542 -214");
```

Comment:
One minor observation
Remember
first argument is the expected result and the second argument is the actual result
Can you change it ?